$NetBSD: patch-aa,v 1.24 2002/06/25 02:13:12 itojun Exp $

use libcrypt before libcrypto, as crypt() in libcrypt is more than just for DES

--- configure.orig	Sat Jun 22 00:56:52 2002
+++ configure	Tue Jun 25 10:54:48 2002
@@ -3943,8 +3943,6 @@
 	CFLAGS="$CFLAGS"
 	;;
 *-*-solaris*)
-	CPPFLAGS="$CPPFLAGS -I/usr/local/include"
-	LDFLAGS="$LDFLAGS -L/usr/local/lib -R/usr/local/lib"
 	need_dash_r=1
 	cat >>confdefs.h <<\_ACEOF
 #define PAM_SUN_CODEBASE 1
@@ -4393,6 +4391,9 @@
 	;;
 esac
 
+# pkgsrc handles any rpath settings this package needs
+need_dash_r=
+
 # Allow user to specify flags
 
 # Check whether --with-cflags or --without-cflags was given.
@@ -6306,6 +6307,10 @@
 #line $LINENO "configure"
 #include "confdefs.h"
 
+#ifdef HAVE_SYS_CDEFS_H
+#include <sys/cdefs.h> 
+#endif
+#include <stdio.h>
 #include <tcpd.h>
 					int deny_severity = 0, allow_severity = 0;
 
@@ -8051,7 +8056,7 @@
 
 
 fi;
-LIBS="$LIBS -lcrypto"
+LIBS="$LIBS -lcrypt -lcrypto"
 cat >conftest.$ac_ext <<_ACEOF
 #line $LINENO "configure"
 #include "confdefs.h"
@@ -17485,7 +17490,7 @@
 echo "                     User binaries: $B"
 echo "                   System binaries: $C"
 echo "               Configuration files: $D"
-echo "                   Askpass program: $E"
+echo "                   Askpass program: ${ASKPASS_PROGRAM}"
 echo "                      Manual pages: $F"
 echo "                          PID file: $G"
 echo "  Privilege separation chroot path: $H"
